docs/blog/2018-07-07-graphic-design-class/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Learning how to code with Gatsby
+date: "2018-07-07"
+author: "Shannon Soper"
+tags: ["college", "case study"]
```

Comment:
design, getting-started

Reply:
do you mean add these to tags or replace current tags with these 2?

Reply:
I have no attachment to current ones, and don't know much about how we're using tags ;)

Reply:
add these in :) take a look at `/blog/tags/`

Reply:
ok done! Thx!